### PR TITLE
Implement POLYGON using OpenGLU's tessellator.

### PR DESCRIPTION
**Pull request by Gleefre**
I finished @Kevinpgalligan's draft.

Further testing is required.

It might be needed to change the `:winding-rule` (for example from `:odd` to `:nonzero`).

### CODE REVIEW

**Comment by Kevinpgalligan** [comment]
Another test: each loop this sketch draws a new polygon with 10 random vertices.

    (defsketch test-polygons
            ((width 100) (height 100))
          (apply #'polygon (loop repeat 20 collect (random width))))

It seems to run indefinitely, so it's pretty robust. And obviously works regardless of whether the polygon is self-intersecting.

File: src/drawing.lisp [hunk 1]
```diff
@@ -109,6 +109,45 @@
...
+    (glu:tess-property tobj :winding-rule :odd)
+    (glu:with-tess-polygon (tobj)
+      (glu:with-tess-contour tobj
+        (loop for (x y) in points
+              ;; Expects 3d coordinates.
+              do (glu:tess-vertex tobj (list x y 0) (list x y)))))
+    (glu:tess-delete tobj)
+    (loop for (primitive . points) in (pt-shapes tobj)
+          do (draw-shape primitive points nil))
+    (draw-shape nil nil points)))
```
Comment (Kevinpgalligan) [inline src/drawing.lisp:123]: Could we maybe leave a comment describing what these expressions are doing? They're a bit mysterious.

Comment (Gleefre) [reply src/drawing.lisp:123]: Sure

**Comment by vydd** [comment]
implemented in #152

**Pull request closed without merging**